mtp_cashbook/assets-src/javascripts/modules/moj.running-total.js

```diff
@@ -0,0 +1,57 @@
+// Running Total module
+// Dependencies: moj, _, jQuery
+
+(function () {
+  'use strict';
+
+  moj.Modules.RunningTotal = {
+    selector: '.js-RunningTotal',
+
+    init: function () {
+      _.bindAll(this, 'render', 'updateTotal');
+      this.cacheEls();
+
+      if (this.$totalsContainer.length) {
+        this.bindEvents();
+        this.render();
+      }
+    },
+
+    cacheEls: function () {
+      this.$items = $('.js-RunningTotal-item');
+      this.$totalsContainer = $(this.selector);
+    },
+
+    bindEvents: function () {
+      moj.Events.on('RunningTotal.render', this.render);
+      this.$items.on('change', this.updateTotal);
+    },
+
+    updateTotal: function () {
+      this.$total.html('&pound;' + this.calculateTotal());
+    },
+
+    calculateTotal: function () {
+      var total = 0;
+
+      this.$items.each(function (i, el) {
+        var $el = $(el);
+        var amount = $el.data('amount');
+
+        if ($el.is(':checked')) {
+          total += parseFloat(amount);
+        }
+      });
+
+      return total.toFixed(2);
+    },
+
+    render: function () {
+      this.$totalsContainer.append('<p>Total payments you have credited: <strong class="js-RunningTotal-amount">&pound;0.00</strong></p>');
```

Comment:
Can we get the markup/text in this passed in from config, and get the text translated?